examples/fuse_examples/imaging/classification/mnist/mnist_data_module.py

```diff
@@ -0,0 +1,124 @@
+from typing import List
+from torch.utils.data import DataLoader
+import pytorch_lightning as pl
+
+from fuse.data.utils.collates import CollateDefault
+from fuseimg.datasets.mnist import MNIST
+from fuse.data.utils.samplers import BatchSamplerDefault
+from fuse.data.utils.split import dataset_balanced_division_to_folds
+
+
+class MNISTDataModule(pl.LightningDataModule):
```

Comment:
Why don't you inherit from the balanced data module?

Reply:
No major reason - it can be done.

note that there is a disadvantage of using the `BalancedDataModule`: we need to create all the datasets in the `__init__` func, while the PL's datamodule create for each stage (in our case only "fit" and "predict").